backend/stakepoold/server.go

```diff
@@ -351,19 +347,83 @@ func (ctx *appContext) processWinningTickets(wt WinningTicketsForBlock) {
 			continue
 		}
 
+		voteCfg, ok := ctx.userVotingConfig[msa]
+		if !ok {
+			// Use defaults if not found.
+			log.Warnf("vote config not found for %v using default",
+				msa)
+			defaultVoteCfg := &userdata.UserVotingConfig{
+				Userid:          0,
+				MultiSigAddress: msa,
+				VoteBits:        ctx.votingConfig.VoteBits,
+				VoteBitsVersion: ctx.votingConfig.VoteVersion,
+			}
+			voteCfg = *defaultVoteCfg
```

Comment:
Why not?
```go
voteCfg = userdata.UserVotingConfig{...
```
But I was wondering if having to do this allocation (`&userdata.UserVotingConfig{`) on every loop is perhaps not great.  This never changes, so it could even be global const.